platform/viewer/cypress/integration/common/OHIFDownloadSnapshotFile.spec.js

```diff
@@ -0,0 +1,152 @@
+describe('OHIF Download Snapshot File', () => {
+  before(() => {
+    cy.openStudy('MISTER^MR');
+    cy.waitDicomImage();
+    cy.expectMinimumThumbnails(5);
+  });
+
+  beforeEach(() => {
+    cy.openDownloadImageModal();
+  });
+
+  afterEach(() => {
+    cy.get('[data-cy="close-button"]')
+      .scrollIntoView()
+      .click();
+  });
+
+  it('checks displayed information for Tablet experience', function() {
+    // Set Tablet resolution
+    cy.viewport(1000, 660);
+    // Visual comparison
+    cy.screenshot('Download Image Modal - Tablet experience');
+    cy.percyCanvasSnapshot('Download Image Modal - Tablet experience');
+  });
+
+  it('checks displayed information for Desktop experience', function() {
+    // Set Desktop resolution
+    cy.viewport(1750, 720);
+    // Visual comparison
+    cy.screenshot('Download Image Modal - Desktop experience');
+    cy.percyCanvasSnapshot('Download Image Modal - Desktop experience');
+    //Check if all elements are displayed
+    cy.get('[data-cy=OHIFModal-header]')
+      .as('downloadImageModal')
+      .should('contain.text', 'Download High Quality Image');
+    // Check input fields
+    cy.get('[data-cy=file-info-container]')
+      .should('contain.text', 'Image width (px)')
+      .should('contain.text', 'Image height (px)')
+      .and('contain.text', 'File name')
+      .and('contain.text', 'File type')
+      .and('contain.text', 'Show Annotations');
```

Comment:
Similarly, by testing that the functionality works as expected, you'll be verifying (to some extent) that all of these fields exist. While the labels are important, losing them would get picked up in a visual test, or an Enzyme/Jest snapshot test